### PR TITLE
EVALG-75: Add Application Design tutorial

### PR DESCRIPTION
**Pull request by Alxe**
### Summary

Add new tutorial for Application Design

### Checks

-   [x] I have updated the documentation accordingly.
-   [x] I have read the [CONTRIBUTING](https://github.com/rticommunity/rticonnextdds-examples/blob/develop/CONTRIBUTING.md) document.

### CODE REVIEW

**Comment by Alxe** [comment]
@alexcamposruiz Please, review this PR.

File: tutorials/application_design/py/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+The example already provides generated types (`VehicleModeling.py`), so nothing
+needs to be done for Python applications.
+
+## How to run
+
+The publisher can be run as a file or module. The expected output contains a
+description of the application. It will end once the vehicle has run out of fuel:
+
+```console
+$ python3.8 -m publisher
```
Comment (alexcamposruiz) [inline tutorials/application_design/py/README.md:20]:
```suggestion
$ python publisher.py
```

File: tutorials/application_design/py/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+This code targets Python 3.8 and uses the Connext Python API.
+
+## How to build
+
+The example already provides generated types (`VehicleModeling.py`), so nothing
+needs to be done for Python applications.
+
+## How to run
+
+The publisher can be run as a file or module. The expected output contains a
```
Comment (alexcamposruiz) [inline tutorials/application_design/py/README.md:16]: For consistency with the rest of this repo, provide the instructions to run as a file only.

File: tutorials/application_design/py/VehicleModeling.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
+# WARNING: THIS FILE IS AUTO-GENERATED. DO NOT MODIFY.
```
Comment (alexcamposruiz) [inline tutorials/application_design/py/VehicleModeling.py:1]: remove warning; replace with (c)

**Review by alexcamposruiz — CHANGES_REQUESTED** [review]

**Review by alexcamposruiz — APPROVED** [review]

**Pull request merged**